src/Interpreters/Context.cpp

```diff
@@ -176,6 +176,12 @@ namespace ErrorCodes
     extern const int NUMBER_OF_COLUMNS_DOESNT_MATCH;
 }
 
+#define SHUTDOWN(log, desc, ptr, method)            \
```

Comment:
Probably you also wrap it `do ... while (false)`, because the absence of `;` at the end of line when you use this macros is a little bit unusual for C++.